samples/templates/default-azuredeploy.json

```diff
@@ -144,9 +144,13 @@
             "FhirServer:Security:EnableAadSmartOnFhirProxy": "[parameters('enableAadSmartOnFhirProxy')]",
             "FhirServer:Security:Authentication:Authority": "[parameters('securityAuthenticationAuthority')]",
             "FhirServer:Security:Authentication:Audience": "[parameters('securityAuthenticationAudience')]",
-            "CosmosDb:ContinuationTokenSizeLimitInKb": "1"
+            "CosmosDb:ContinuationTokenSizeLimitInKb": "1",
+            "SqlServer:Initialize": "true",
+            "DataStore": "[if(equals(parameters('solutionType'),'FhirServerCosmosDB'), 'CosmosDb', 'SqlServer')]"
         },
-        "combinedFhirServerConfigProperties": "[union(variables('staticFhirServerConfigProperties'), parameters('additionalFhirServerConfigProperties'))]"
+        "combinedFhirServerConfigProperties": "[union(variables('staticFhirServerConfigProperties'), parameters('additionalFhirServerConfigProperties'))]",
+        "computedSqlServerReference": "[concat('Microsoft.Sql/servers/', variables('serviceName'))]",
+        "sqlAdminPass" : "[concat(uniqueString(resourceGroup().id, deployment().name, 'ac45f99e-035e-4712-941d-cafac13536ba'), 'x', '!')]"
```

Comment:
This can be reverse-engineered. It should be passed in as a parameter I think (since we don't necessarily want to change it with each deployment).